images/awx-ee/requirements.txt

```diff
@@ -6,3 +6,4 @@ ansible-lint
 flake8
 yamllint
 ruamel.yaml
+pygithub
```

Comment:
All those other changes seem to be from a newer version of ansible-builder. This was the one change I actually made.